client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -513,6 +543,7 @@ class BusinessDetails extends Component {
 				{ ( { getInputProps, handleSubmit, values, isValidForm } ) => {
 					// Show extensions when the currently selling elsewhere checkbox has been answered.
 					const showExtensions = values.selling_venues !== '';
+					const otherPlatform = getInputProps( 'other_platform' );
```

Comment:
Was this created for a specific reason?  Can we move this directly inside the `SelectControl ` props below?

`{ ...getInputProps( 'other_platform' ) }`

Reply:
Yes. The constant `otherPlatform` is used to check if the user selected the option `other` to show the input text or not. The reason to declare this constant is to not call the function `getInputProps( 'other_platform' )` twice.

Reply:
Ah! I see.  In that case we may want to check using `values` and then we can remove this constant.  Will add a comment below.

Reply:
I didn't see it. Good idea!